src/index.html

```diff
@@ -13,6 +13,39 @@
     />
```

Comment:
The link to the stylesheet should be a CSS file, not a SCSS file. Please compile your SCSS to CSS and link the CSS file here.